src/Asset/Capacity/AbstractCapacity.php

```diff
@@ -65,6 +65,45 @@ public function getSpecificRights(): array
         return [];
     }
 
+    public function isUsed(string $classname): bool
+    {
+        return (new $classname())->getDefinition()->hasCapacityEnabled($this);
+    }
+
+    protected function countTargetItemUsed(
+        string $source_itemtype,
+        string $target_itemtype,
+        ?string $distinctField = null
+    ): int {
```

Comment:
This method lack PHP doc (same for countSourceItemCapacityUsages and countAssets)